tokens/src/themes/light/components/Carousel.json

```diff
@@ -1,43 +1,44 @@
 {
   "color": {
+    "$type": "color",
     "carousel": {
       "control": {
-        "base": { "value": "{color.white}", "type": "color", "source": "$carousel-control-color" }
+        "base": { "source": "$carousel-control-color", "$value": "{color.white}" }
       },
       "indicator": {
-        "active-bg": { "value": "{color.white}", "type": "color", "source": "$carousel-indicator-active-bg" }
+        "active-bg": { "source": "$carousel-indicator-active-bg", "$value": "{color.white}" }
       },
-      "caption": { "value": "{color.white}", "type": "color", "source": "$carousel-caption-color" }
+      "caption": { "source": "$carousel-caption-color", "$value": "{color.white}" }
     }
   },
   "content": {
+    "$type": "file",
     "carousel": {
       "control": {
         "bg": {
           "prev-icon": {
-            "value": "url(\"data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' fill='{color.carousel.control.base}' viewBox='0 0 8 8'%3e%3cpath d='M5.25 0l-4 4 4 4 1.5-1.5-2.5-2.5 2.5-2.5-1.5-1.5z'/%3e%3c/svg%3e\")",
-            "type": "file",
             "source": "$carousel-control-prev-icon-bg",
             "outputReferences": false,
-            "modify": [{ "type": "str-replace", "toReplace": "#", "replaceWith": "%23" }]
+            "modify": [{ "type": "str-replace", "toReplace": "#", "replaceWith": "%23" }],
+            "$value": "url(\"data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' fill='{color.carousel.control.base}' viewBox='0 0 8 8'%3e%3cpath d='M5.25 0l-4 4 4 4 1.5-1.5-2.5-2.5 2.5-2.5-1.5-1.5z'/%3e%3c/svg%3e\")"
```

Comment:
I'm curious about the hardcoded `%3c` for `<` and `%3e` for `>` here. I'm not 100% sure how `str-replace` in `modify` works, could that be used for those similarly to how it's being used for `#`?

Reply:
This is a [modifier](https://github.com/openedx/paragon/blob/alpha/tokens/style-dictionary.js#L112) for replacing one value in a string with another. After playing with this modifier a bit, I realized that it doesn't support multiple replacement parameters, for example, you can't use something like:

```
            "modify": [
              { "type": "str-replace", "toReplace": "#", "replaceWith": "%23" },
              { "type": "str-replace", "toReplace": ">", "replaceWith": "%3E" },
              { "type": "str-replace", "toReplace": "<", "replaceWith": "%3C" }
            ],
            "$value": "url(\"data:image/svg+xml,<svg xmlns='http://www.w3.org/2000/svg' fill='{color.carousel.control.base}' viewBox='0 0 8 8'><path d='M2.75 0l-1.5 1.5 2.5 2.5-2.5 2.5 1.5 1.5 4-4-4-4z'/></svg>\")"
```

I'm not sure if we need to handle any cases right now other than the `#` symbol (we're currently replacing only that symbol), but I think the option of multiple parameters for replacement would be useful in the future 💯

Reply:
That's unfortunate. It would be really nice to support multiple `str-replace`s. Not a dealbreaker though.